execution-order/execution-order.css

```diff
@@ -0,0 +1,58 @@
+html, body {
+  margin: 0;
+  padding: 0;
+}
+#main {
+  display: grid;
+  grid-template-columns: min-content minmax(0, 1fr);
+  height: 100vh;
+}
+#main.ready {
+  animation: slidein .25s ease-in-out;
+}
+.closer {
+  max-height: 100vh;
+}
+.closer::after {
+  content: "\bb";
+}
+ol {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  font-size: 16px;
+  overflow-y: auto;
+}
+li {
+  position: relative;
+  user-select: none;
+  border: 1px solid silver;
+  border-left: 0;
+  padding: 0.3em 0;
+  background: white;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  cursor: grab;
+}
+li:not(:first-child) {
+  margin-top: -1px;
+}
+.dragger::before {
+  content: "\2261"
```

Comment:
@narcolepticinsomniac Currently the drag indicator is just a unicode char [2261](https://www.compart.com/en/unicode/U+2261). We will need an SVG to adjust width/height and to make it cross browsers.

Reply:
Why do we need any indicator/icon at all?

Reply:
To indicate users that the list can be dragged?

Reply:
If you use `cursor: move` like I suggested, the action would be pretty much obvious. Since the entire row is draggable, there's no *need* for a separate icon.